x/incentives/types/gauge.go

```diff
@@ -31,12 +31,12 @@ func NewGauge(id uint64, isPerpetual bool, distrTo lockuptypes.QueryCondition, c
 
 // IsUpcomingGauge returns true if the gauge's distribution start time is after the provided time.
 func (gauge Gauge) IsUpcomingGauge(curTime time.Time) bool {
-	return curTime.After(gauge.StartTime)
+	return curTime.Before(gauge.StartTime) || curTime.Equal(gauge.StartTime)
```

Comment:
I believe active gauge instead of upcoming should be inclusive of curTime, based on:

https://github.com/osmosis-labs/osmosis/blob/dd0c5a7c2ca253cac493991dac438806ae2527b2/x/incentives/keeper/distribute.go#L51-L53

Reply:
Agreed